release/changes.md

```diff
@@ -1 +1 @@
-- [NEW] TBD
\ No newline at end of file
+- [FIX] API requests do allow time for build scans to become available
```

Comment:
```suggestion
- [FIX] Wait up to 10 seconds for Build Scan data to become available
```

Reply:
```suggestion
- [FIX] The Build Validation Scripts do not wait long enough for Build Scan data to become available.
```

Reply:
I actually missed a word, it should read:

```
- [FIX] API requests do not allow time for build scans to become available
```

This is how we typically write fixes in the release notes, by describing the problem rather than the solution.